#! /bin/sh /usr/share/dpatch/dpatch-run
## 07-autoconf-fix-gperf.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fix an autotools error in GPERF handling.

@DPATCH@
--- ACE_wrappers.orig/m4/ace.m4
+++ ACE_wrappers/m4/ace.m4
@@ -535,7 +535,7 @@
      AC_MSG_WARN([existing gperf may be overwritten during installation])
     ],[])
  fi
- AM_CONDITIONAL([COMPILE_GPERF], [test X$ace_user_enable_gperf = Xyes])
+ AM_CONDITIONAL([BUILD_GPERF], [test X$ace_user_enable_gperf = Xyes])
 
  ACE_ENABLE_QOS
  ACE_ENABLE_SSL
